semantic/src/site/globals/site.overrides

```diff
@@ -123,6 +120,66 @@ p strong {
   margin-left: 10px;
 }
 
+/*--------------------------------
+   Application home & create
+---------------------------------*/
+#application-area {
+  display: flex;
+  flex-direction: column;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+#application-area .top-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+#application-area .company-title {
+  width: 100%;
```

Comment:
I don't think the .top-container needs to have flex definitions. The natural vertical flow of a straight "div" is enough. Then you won't need the "width" property in .company-title.

Reply:
Done: 466b283